app/controllers/api_controller/logger.rb

```diff
@@ -24,7 +24,7 @@ def log_api_auth
 
     def log_api_request
       @parameter_filter ||= ActionDispatch::Http::ParameterFilter.new(Rails.application.config.filter_parameters)
-      log_request("Request", @req)
+      log_request("Request", @req.to_hash)
```

Comment:
This seems potentially expensive if the logging is disabled or moved to a lower level because you still have to generate the hash.